go.sum

```diff
@@ -2152,6 +2152,7 @@ go.uber.org/atomic v1.11.0 h1:ZvwS0R+56ePWxUNi+Atn9dWONBPp/AUETXlHW0DxSjE=
 go.uber.org/atomic v1.11.0/go.mod h1:LUxbIzbOniOlMKjJjyPfpl4v+PKK2cNJn91OQbhoJI0=
 go.uber.org/goleak v1.1.10/go.mod h1:8a7PlsEVH3e/a/GLqe5IIrQx6GzcnRmZEufDUTk4A7A=
 go.uber.org/goleak v1.1.12/go.mod h1:cwTWslyiVhfpKIDGSZEM2HlOvcqm+tG4zioyIeLoqMQ=
```

Comment:
it looks like `go mod tidy` hasn't been run for a while (duplicated module entries for different versions)  - can you please?

Reply:
I did run go mod tidy in this PR but I guess someone added some stuff on the main branch without running go mod tidy during that time. Now its merged, sorry I did not see this sooner :cry: